hector_mapping/src/HectorMappingRos.cpp

```diff
@@ -92,6 +92,10 @@ HectorMappingRos::HectorMappingRos()
   private_nh_.param("output_timing", p_timing_output_,false);
 
   private_nh_.param("map_pub_period", p_map_pub_period_, 2.0);
+  // Offset (ROS Duration) to add to pose data when publishing it. A
+  // positive value shifts the pose ts to the future and enables using
+  // move base to move around while mapping.
+  private_nh_.param("pose_ts_offset", p_pose_ts_offset_, 1.0);
```

Comment:
Sorry, one more thing, use a name that can be understood like `transform_tolerance`.

```suggestion
  private_nh_.param("transform_tolerance", p_pose_ts_offset_, 1.0);
```